templates/strict_authoritative/bind/named.conf.local.j2

```diff
@@ -0,0 +1,119 @@
+// BIND 9 Configuration - named.conf.local - generated by systemli.bind9 ansible role
+// DO NOT edit, change your ansible config and re-run your ansible playbooks
+//
+
+// Consider adding the 1918 zones here, if they are not used in your
+// organization
+//include "/etc/bind/zones.rfc1918";
+
+{% if bind9_statistics_enabled %}
+
+statistics-channels {
+  inet 127.0.0.1 port 8053 allow { 127.0.0.1; };
+};
+{% endif %}
+{% if bind9_masters|default() %}
+{%   for master in bind9_masters %}
+masters "{{ master.name }}" {
+{%     for addr in master.addresses %}
+    {{ addr }};
+{%     endfor %}
+};
+{%   endfor %}
+{% endif %}
+{% if bind9_masters_extra|default() %}
+{%   for master in bind9_masters_extra %}
+masters "{{ master.name }}" {
+{%     for addr in master.addresses %}
+    {{ addr }};
+{%     endfor %}
+};
+{%   endfor %}
+{% endif %}
+{% if bind9_acl is defined %}
+
+	// Custom acls
+{%   for acl_item in bind9_acl %}
+acl "{{ acl_item.name }}" {
+{%     for acl_address in acl_item.addresses %}
+	{{ acl_address }};
+{%     endfor %}
+};
+{%   endfor %}
+{% endif %}
+
+// The following zones are managed by this DNS Server //
+{% for zone in (bind9_zones_static + bind9_zones_dynamic)|sort(attribute='name') %}
+{% set zone_type = zone.type|default(bind9_zone_type|default('master')) %}
+zone "{{ zone.name }}" {
+    type {{ zone_type }};
+{%   if zone_type == 'master' %}
+    file "/etc/bind/zones/db.{{ zone.name }}";
+    allow-query {
+{%     if zone.allow_query is defined and zone.allow_query | length() > 0  %}
+{%       for allow_query_item in zone.allow_query | list %}
+        {{ allow_query_item }};
+{%       endfor %}
+    };
+{%     else %}
+        any;
+    };
+{%     endif %}
+{%     set zone_secondaries = zone.secondaries | default( bind9_secondaries | default( [] ) ) %}
```

Comment:
(BUG) All of these "secondaries" references must be changed to "slaves" for the role to work

Reply:
Initially I introduced "primary" and "secondary", as [Bind reference documentation states that's the new wording](https://bind9.readthedocs.io/en/latest/reference.html?highlight=primary%20master#type), but it's in a newer version that the one shipped with present debian stable, so I reverted it, but forgot that.